infra/app/functions.bicep

```diff
@@ -0,0 +1,72 @@
+param functionAppName string
+param location string = resourceGroup().location
+param hostingPlanId string
+param storageAccountName string
+@secure()
+param instrumentationKey string
+@secure()
+param openAIEndpoint string

```

Comment:
Should be "openAi" across the board, make sure you use consistent casing

Reply:
Modified.